.github/workflows/update-server-side-sdk-schema.yml

```diff
@@ -38,23 +35,42 @@ on:
         description: 'Path to the OpenAPI examples.'
         required: true
         type: string
+      appId:
+        type: string
+        required: false
+        description: 'GitHub app id for release process.'
     secrets:
-      GITHUB_TOKEN:
+      GH_TOKEN:
         description: 'GitHub token with permissions to create releases and perform other necessary operations.'
         required: false
+      APP_PRIVATE_KEY:
```

Comment:
Which rights we can't provide with token that we need an app?

Reply:
Let's try it with the default github token without app.